django_backend/tako_CRM/tako_CRM/settings.py

```diff
@@ -0,0 +1,125 @@
+"""
+Django settings for tako_CRM project.
+
+Generated by 'django-admin startproject' using Django 3.2.4.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.2/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-d2kmoh#6g%wm58ji2=%v@imakjs63$acu59-04^3z3+bei*x+5'
```

Comment:
It isn't a problem to upload the secret key, as long you use a different one for Production (when you deploy your application). What you can also do, if you're interested, is create two files: `.env` and `.env.example` and fill it with some enviroment variables and then just replace this line of code with something like:
`SECRET_KEY = os.getenv('SECRET_KEY')`

And these suggestions can be used throughout the configuration file, whether is the Debug variable, Allowed Hosts etc.

Reply:
A new issue dedicated to this was created as a future enhancement: https://github.com/atosborges00/tako-CRM/issues/8